phpunit/class-wp-theme-json-test.php

```diff
@@ -978,6 +978,11 @@ function test_remove_insecure_properties_removes_unsafe_styles() {
 						),
 					),
 					'blocks'   => array(
+						'core/image'  => array(
+							'filter' => array(
+								'duotone' => 'var:preset|duotone|blue-red',
```

Comment:
At the moment, this will be ignored because we haven't taught kses to allow this. We can add the code to make sure this passes in a follow-up PR.

Reply:
Is this notation something we already support? Why not just use CSS directly?

Reply:
Yeah. This is how the user provided styles are stored in the block attributes as well as in the CPT for global styles.